src/raw/config-avalanche.js

```diff
@@ -0,0 +1,201 @@
+config = {
+  graphEndpoint:
+    "https://api.thegraph.com/subgraphs/name/float-capital/float-capital-avalanche",
```

Comment:
i'm not certain whether we need to have graph endpoints here?

just cause they're liable to change / perhaps be unstable.

Similar for block explorer / rpc endpoint / bridgeLink seems like users can provide this themselves.

think we could possibly slim down the config a bit to keep it more relevant but let me know your thoughts not averse to expanding it.

Reply:
The non-float specific stuff like RPC endpoint etc aren't necessary but it's kinda useful to have and it doesn't hurt to have them in here so I reckon we keep them. Also is useful in our code because then we need less config there 'cause it's already here.

The graph endpoints I definitely think should be in here. Trying to figure out which graph endpoint is the correct one from a user's perspective is a pain in the butt.

Reply:
Yeah, it is debatable. But the graph is pretty stable now, so I think this should be generally fine.